docs/question.md

```diff
@@ -374,10 +364,33 @@ int m = 4;
 </pl-code>
 ```
 
-## Escaping `<markdown>` tags
+### Escaping `<markdown>` tags
 
 Under the hood, PrairieLearn is doing some very simple parsing to determine what pieces of a question to process as Markdown: it finds an opening `<markdown>` tag and processes everything up to the closing `</markdown>` tag. But what if you want to have a literal `<markdown>` or `</markdown>` tag in your question? PrairieLearn defines a special escape syntax to enable this. If you have `<markdown#>` or `</markdown#>` in a Markdown block, they will be rendered as `<markdown>` and `</markdown>` respectively (but will not be used to find regions of text to process as Markdown). You can use more hashes to produce different strings: for instance, to have `<markdown###>` show up in the output, write `<markdown####>` in your question.
 
+## Using LaTeX in questions
+
+PrairieLearn supports LaTeX equations in questions. You can view a full list of supported MathJax commands [here](https://docs.mathjax.org/en/latest/input/tex/macros/index.html).
+
+Inline equations can be written using `$x^2$`, and display equations can be written using `$$x^2$$` or `\[x^2\]`. For example:
+
+```html title="question.html"
+<p>Here is some inline math: $x^2$. Here is some display math: $$x^2$$</p>
+<p>What is the total force $F$ currently acting on the particle?</p>
+```
+
+It can also be used inside `<markdown>` elements:
+
+<!-- prettier-ignore -->
+```html title="question.html"
+<markdown>
+  $$\phi = \frac{1+\sqrt{5}}{2}$$
+
+  Need to use `$` outside of math-mode? Place inside inline code.
+  So, excel formulas would look like: `$A$1`, `B$5`, `$C9`, `D2`.
```

Comment:
The issue with this is that it will change the `$` in different formatting, which is now always expected. You can also use `\$`, though this may be tricky in markdown (hopefully fixed by the followup to #10570 for server-side). Another option is to use `<span class="mathjax_ignore">$34.45</span>`. See https://prairielearn.readthedocs.io/en/latest/faq/#how-can-i-use-a-dollar-sign-without-triggering-math-mode.